### ISSUE TITLE
Multiple-search not working

### ISSUE THREAD

**Issue by kurisukun** [wenzhixin/bootstrap-table#3753]
Hi guys. Thx for your work, it's helping me for a project. however I don't understand the extension "multiple earch because when i follow the "tutorial" i.e.:

including this in the page of my table: 
<script src="extensions/multiple-search/bootstrap-table-multiple-search.js"></script>

and of course creating the "bootstrap-table-multiple-search.js" file in the right directory. 

I found this fiddle which works perfectly but without using the extension: http://jsfiddle.net/aq9Laaew/14340/

And when I try to use the extension, it doesn't work (here the example is without an external file):
http://jsfiddle.net/aq9Laaew/14354/

I think I missed something, can you please help me?

**Comment by Ka7ut0** [comment]
I faced the same problem as you. After fiddling with it, I ended up grabbing the whole JS code block from Fiddler and save it in a separate file extensions/multiple-search/bootstrap-table-multiple-search-1_11_1.js

When you compare it to the bootstrap-table-multiple-search.js provided in the extensions, you'll see how much more code the Fiddler version has.

Maybe the wrong file is provided or we need a dependency?

![screenshot_20190306_110253](https://user-images.githubusercontent.com/47562131/53872754-2d41ad00-3fff-11e9-99a4-d3f129634460.png)

Anyway, using this one, everything works as expected: https://pastebin.com/fPVXM9Hu
